src/ui/scss/component/_button.scss

```diff
@@ -94,6 +94,16 @@
   align-items: flex-start;
 }
 
+// Quick fix because this is a pain
+// There is something weird with wrapping buttons. Some places we want to wrap and others we want to ellips
```

Comment:
Good job noting why the rules are being broken so a future dev and/or future you can understand